packages/labs/context/rollup.config.js

```diff
@@ -0,0 +1,12 @@
+/**
+ * @license
+ * Copyright 2021 Google LLC
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import {litProdConfig} from '../../../rollup-common.js';
+
+export default litProdConfig({
+  classPropertyPrefix: 'Ϋ',
```

Comment:
We moved away from greek prefixes. See the common Rollup config.

Reply:
I think I've updated this correctly, let me know if not.